package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@financial-times/content-tree",
+	"name": "@content-tree/types",
```

Comment:
When we publish these to npm, do we have a private npm space for FT stuff (i forget the current situation)? If not, is that namespace (@content-tree) gonna be too generic?

Reply:
we don't have a _private_ npm space (like, we publish public npm packages under our owned org namespces `@financial-times` and `@dotcom-reliability-kit`). but we do also own the `@content-tree` namespace (nobody can publish to it but us) and could use it. 
just because it's less of a horribly long name to have `@content-tree/json-schema-transit` than `@financial-times/content-tree-json-schema-transit`. 

this repo isn't actually publishing anything yet, even in any of these PRs, so we have plenty of time to chnge our minds